src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ScrollViewer.cs

```diff
@@ -246,7 +241,12 @@ public async Task When_ScrollViewer_Centered_With_Margin_Inside_Tall_Rectangle()
 
 			const double ScrollViewerHeight = ContentHeight + 2 * ContentMargin;
 			await WindowHelper.WaitForEqual(ScrollViewerHeight, () => SUT.ActualHeight);
-			Assert.AreEqual(ContentHeight, SUT.ExtentHeight);
+
+			// Issue needs to be fixed first for WASM for Right and Bottom Margin missing
+			// Details here: https://github.com/unoplatform/uno/issues/7000
+#if !__WASM__
+			Assert.AreEqual(ScrollViewerHeight, SUT.ExtentHeight);
```

Comment:
@dr1rrb and @MartinZikmund My PR is already merged since last week-end. @MartinZikmund maybe you can in your PR ignore the whole test on WASM but make sure to also move the comments I left next to the ignore platform please.